application/client.core/src/app/environment/components/sidebar/search.manager/providers/provider.ts

```diff
@@ -67,12 +67,14 @@ export abstract class Provider<T> {
         edit: Subject<string | undefined>,
         context: Subject<IContextMenuEvent>,
         doubleclick: Subject<IDoubleclickEvent>,
+        update: Subject<void>,
     } = {
         change: new Subject(),
         selection: new Subject(),
         edit: new Subject(),
         context: new Subject(),
         doubleclick: new Subject(),
+        update: new Subject(),
```

Comment:
It isn't clear, why we need the addition event `update` as soon as we already have the event `change`?